irmaclient/client.go

```diff
@@ -589,14 +589,24 @@ func (client *Client) credCandidates(request irma.SessionRequest, con irma.Attri
 			// then the entire conjunction is unsatisfiable
 			satisfiable = false
 		}
-		if len(c) == 0 {
-			if client.addCredSuggestion(request, credTypeID) {
-				// No acceptable credentials found. Excluding some nonsensical cases,
-				// add an "empty" credential (i.e. without hash) as a suggestion to the user
-				c = append(c, &credCandidate{Type: credTypeID})
+		// Determine whether we force an attribute value for any attribute requested from this credential here.
```

Comment:
To me the "we" makes this comment confusing: it is not "us" but the requestor that requests specific attribute values. I had to read the code below in order to be able to understand the comment, instead of vice versa. Suggestion: `Determine if the session request forces (...)`

Reply:
fixed it, good suggestion